exporters/prometheus/test/prometheus_exporter_utils_test.cc

```diff
@@ -469,3 +470,9 @@ TEST(PrometheusExporterUtils, TranslateToPrometheusMultipleAggregators)
                prometheus_client::MetricType::Gauge, 1, vals);
 }
 OPENTELEMETRY_END_NAMESPACE
+#else
```

Comment:
Yeah, you wouldn't have needed it if you excluded the entire folder. It would've been just 2-3 files change instead of 51.